### DISCUSSION TITLE
Github Output Parameters

### DISCUSSION THREAD

**Discussion by teaguenology** [crowdin/github-action#203] [Q&A] [answered]
I'm using the github action to download translations and create a pull request to merge them into the default branch.

I wanted to follow up the PR creation step with a slack message containing the PR's url. 

I'm not seeing any outputs defined in the action.yml or documentation. I do see the url get logged in the github action output.

Is there a way for me to pull the url and use it in a subsequent action?

**Comment by andrii-bodnar** [comment]
@teaguenology thanks for your request! It would be nice to have the `pull_request_url` output in this Action. Converting this discussion into an enhancement issue.

**Comment by andrii-bodnar** [comment, marked as answer]
Available in [v1.15.0](https://github.com/crowdin/github-action/releases/tag/v1.15.0)

**Reply by teaguenology** [reply]
Thank you!
